### DISCUSSION TITLE
Homebrew  formula

### DISCUSSION THREAD

**Discussion by matiaskorhonen** [transifex/cli#66] [Ideas]
It would be great if you could add a Homebrew [formula](https://docs.brew.sh/Formula-Cookbook) for the client, this would make installation and updates on macOS a lot nicer.

The client doesn't currently meet the criteria for [Acceptable Formulae](https://docs.brew.sh/Acceptable-Formulae) due to the GitHub repository having too few stars, watchers, or forks (‘<30 forks, <30 watchers and <75 stars’) but installation via a [tap would still be possible](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap).

Example formula for Transifex CLI v1.1.0:

```rb
class Transifex < Formula
  desc "Command-line client for Transifex"
  homepage "https://developers.transifex.com/docs/cli"
  url "https://github.com/transifex/cli/archive/refs/tags/v1.1.0.tar.gz"
  sha256 "2aacd1ac5efbe15cc77fd5732efd724c5a3b2c4f288a441cea96344893662bde"
  license "Apache-2.0"
  head "https://github.com/transifex/cli.git", branch: "devel"

  depends_on "go" => :build

  def install
    bin.mkpath
    system "go", "build", *std_go_args(ldflags: "-X 'github.com/transifex/cli/internal/txlib.Version=#{version}'"), "-o", bin/"tx"
  end

  test do
    assert_match "TX Client, version=#{version}", shell_output("#{bin}/tx -v")
  end
end
```

**Comment by guruz** [comment]
I think this exists now.

https://formulae.brew.sh/formula/transifex-cli

```
brew install transifex-cli
```
